[2023-08-30T18:56:37.302+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T18:56:37.320+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-25T00:00:00+00:00 [queued]>
[2023-08-30T18:56:37.327+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-25T00:00:00+00:00 [queued]>
[2023-08-30T18:56:37.328+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T18:56:37.343+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-25 00:00:00+00:00
[2023-08-30T18:56:37.352+0000] {standard_task_runner.py:57} INFO - Started process 503 to run task
[2023-08-30T18:56:37.359+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-25T00:00:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmp546nvd8g']
[2023-08-30T18:56:37.363+0000] {standard_task_runner.py:85} INFO - Job 8: Subtask install_required_packages
[2023-08-30T18:56:37.412+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T18:56:37.538+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-25T00:00:00+00:00 [running]> on host 5e7fbf8b7901
[2023-08-30T18:56:37.578+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T18:56:37.709+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-25T00:00:00+00:00'
[2023-08-30T18:56:37.716+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T18:56:37.721+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy']
[2023-08-30T18:56:37.737+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T18:56:39.293+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T18:56:40.031+0000] {subprocess.py:93} INFO - Collecting sodapy
[2023-08-30T18:56:40.111+0000] {subprocess.py:93} INFO -   Downloading sodapy-2.2.0-py2.py3-none-any.whl (15 kB)
[2023-08-30T18:56:40.128+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T18:56:40.134+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T18:56:40.135+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T18:56:40.150+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T18:56:40.155+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T18:56:43.792+0000] {subprocess.py:93} INFO - Installing collected packages: sodapy
[2023-08-30T18:56:43.885+0000] {subprocess.py:93} INFO - Successfully installed sodapy-2.2.0
[2023-08-30T18:56:44.246+0000] {subprocess.py:93} INFO - 
[2023-08-30T18:56:44.248+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T18:56:44.250+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T18:56:44.623+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T18:56:44.660+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230825T000000, start_date=20230830T185637, end_date=20230830T185644
[2023-08-30T18:56:44.709+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T18:56:44.739+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:19:23.803+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:19:23.813+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-25T00:00:00+00:00 [queued]>
[2023-08-30T19:19:23.824+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-25T00:00:00+00:00 [queued]>
[2023-08-30T19:19:23.825+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:19:23.840+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-25 00:00:00+00:00
[2023-08-30T19:19:23.848+0000] {standard_task_runner.py:57} INFO - Started process 217 to run task
[2023-08-30T19:19:23.852+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-25T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmp4t_e64r7']
[2023-08-30T19:19:23.856+0000] {standard_task_runner.py:85} INFO - Job 5: Subtask install_required_packages
[2023-08-30T19:19:23.922+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:19:24.018+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-25T00:00:00+00:00 [running]> on host 884d86529f65
[2023-08-30T19:19:24.046+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:19:24.198+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-25T00:00:00+00:00'
[2023-08-30T19:19:24.200+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:19:24.202+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy']
[2023-08-30T19:19:24.218+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:19:28.611+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T19:19:29.507+0000] {subprocess.py:93} INFO - Collecting sodapy
[2023-08-30T19:19:29.593+0000] {subprocess.py:93} INFO -   Downloading sodapy-2.2.0-py2.py3-none-any.whl (15 kB)
[2023-08-30T19:19:29.604+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T19:19:29.616+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T19:19:29.617+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T19:19:29.618+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T19:19:29.620+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T19:19:57.621+0000] {subprocess.py:93} INFO - Installing collected packages: sodapy
[2023-08-30T19:19:57.677+0000] {subprocess.py:93} INFO - Successfully installed sodapy-2.2.0
[2023-08-30T19:19:57.809+0000] {subprocess.py:93} INFO - 
[2023-08-30T19:19:57.811+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T19:19:57.812+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T19:19:58.167+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:19:58.268+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230825T000000, start_date=20230830T191923, end_date=20230830T191958
[2023-08-30T19:19:58.341+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:19:58.369+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:31:06.369+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:31:06.378+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-25T00:00:00+00:00 [queued]>
[2023-08-30T19:31:06.392+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-25T00:00:00+00:00 [queued]>
[2023-08-30T19:31:06.393+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:31:06.415+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-25 00:00:00+00:00
[2023-08-30T19:31:06.430+0000] {standard_task_runner.py:57} INFO - Started process 220 to run task
[2023-08-30T19:31:06.455+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-25T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpleiswvar']
[2023-08-30T19:31:06.461+0000] {standard_task_runner.py:85} INFO - Job 6: Subtask install_required_packages
[2023-08-30T19:31:06.514+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:31:06.618+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-25T00:00:00+00:00 [running]> on host f6bcf103ff79
[2023-08-30T19:31:06.665+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:31:06.793+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-25T00:00:00+00:00'
[2023-08-30T19:31:06.798+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:31:06.801+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy']
[2023-08-30T19:31:06.817+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:31:08.988+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T19:31:11.135+0000] {subprocess.py:93} INFO - Collecting sodapy
[2023-08-30T19:31:11.268+0000] {subprocess.py:93} INFO -   Downloading sodapy-2.2.0-py2.py3-none-any.whl (15 kB)
[2023-08-30T19:31:11.602+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T19:31:11.633+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T19:31:11.643+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T19:31:11.646+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T19:31:11.647+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T19:31:33.875+0000] {subprocess.py:93} INFO - Installing collected packages: sodapy
[2023-08-30T19:31:33.931+0000] {subprocess.py:93} INFO - Successfully installed sodapy-2.2.0
[2023-08-30T19:31:33.940+0000] {subprocess.py:93} INFO - 
[2023-08-30T19:31:33.942+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T19:31:33.943+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T19:31:34.112+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:31:34.185+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230825T000000, start_date=20230830T193106, end_date=20230830T193134
[2023-08-30T19:31:34.227+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:31:34.249+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:41:18.495+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:41:18.507+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-25T00:00:00+00:00 [queued]>
[2023-08-30T19:41:18.519+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-25T00:00:00+00:00 [queued]>
[2023-08-30T19:41:18.521+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:41:18.539+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-25 00:00:00+00:00
[2023-08-30T19:41:18.548+0000] {standard_task_runner.py:57} INFO - Started process 276 to run task
[2023-08-30T19:41:18.551+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-25T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpb0izuwb0']
[2023-08-30T19:41:18.557+0000] {standard_task_runner.py:85} INFO - Job 5: Subtask install_required_packages
[2023-08-30T19:41:18.624+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:41:18.747+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-25T00:00:00+00:00 [running]> on host 81eeb4b7c8a9
[2023-08-30T19:41:18.774+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:41:18.878+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-25T00:00:00+00:00'
[2023-08-30T19:41:18.895+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:41:18.897+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy']
[2023-08-30T19:41:18.926+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:41:25.207+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T19:41:33.496+0000] {subprocess.py:93} INFO - Collecting sodapy
[2023-08-30T19:41:33.830+0000] {subprocess.py:93} INFO -   Downloading sodapy-2.2.0-py2.py3-none-any.whl (15 kB)
[2023-08-30T19:41:33.848+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T19:41:33.868+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T19:41:33.869+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T19:41:33.870+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T19:41:33.871+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T19:41:49.410+0000] {subprocess.py:93} INFO - Installing collected packages: sodapy
[2023-08-30T19:41:49.477+0000] {subprocess.py:93} INFO - Successfully installed sodapy-2.2.0
[2023-08-30T19:41:49.487+0000] {subprocess.py:93} INFO - 
[2023-08-30T19:41:49.489+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T19:41:49.490+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T19:41:49.778+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:41:49.834+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230825T000000, start_date=20230830T194118, end_date=20230830T194149
[2023-08-30T19:41:49.886+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:41:49.912+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:50:05.876+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:50:05.896+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-25T00:00:00+00:00 [queued]>
[2023-08-30T19:50:05.914+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-25T00:00:00+00:00 [queued]>
[2023-08-30T19:50:05.916+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:50:05.949+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-25 00:00:00+00:00
[2023-08-30T19:50:05.954+0000] {standard_task_runner.py:57} INFO - Started process 243 to run task
[2023-08-30T19:50:05.996+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-25T00:00:00+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpi3983rhs']
[2023-08-30T19:50:06.359+0000] {standard_task_runner.py:85} INFO - Job 2: Subtask install_required_packages
[2023-08-30T19:50:10.949+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:50:11.064+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-25T00:00:00+00:00 [running]> on host 206ecb1dc8df
[2023-08-30T19:50:11.118+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:50:11.364+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-25T00:00:00+00:00'
[2023-08-30T19:50:11.399+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:50:11.410+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy geoalchemy2']
[2023-08-30T19:50:11.432+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:50:25.609+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T19:50:30.705+0000] {subprocess.py:93} INFO - Collecting sodapy
[2023-08-30T19:50:30.866+0000] {subprocess.py:93} INFO -   Downloading sodapy-2.2.0-py2.py3-none-any.whl (15 kB)
[2023-08-30T19:50:31.234+0000] {subprocess.py:93} INFO - Collecting geoalchemy2
[2023-08-30T19:50:31.255+0000] {subprocess.py:93} INFO -   Downloading GeoAlchemy2-0.14.1-py3-none-any.whl (54 kB)
[2023-08-30T19:50:31.302+0000] {subprocess.py:93} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 55.0/55.0 kB 1.4 MB/s eta 0:00:00
[2023-08-30T19:50:31.313+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T19:50:31.323+0000] {subprocess.py:93} INFO - Requirement already satisfied: SQLAlchemy>=1.4 in /home/***/.local/lib/python3.8/site-packages (from geoalchemy2) (1.4.49)
[2023-08-30T19:50:31.324+0000] {subprocess.py:93} INFO - Requirement already satisfied: packaging in /home/***/.local/lib/python3.8/site-packages (from geoalchemy2) (23.1)
[2023-08-30T19:50:31.330+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T19:50:31.337+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T19:50:31.342+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T19:50:31.358+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T19:50:31.431+0000] {subprocess.py:93} INFO - Requirement already satisfied: greenlet!=0.4.17 in /home/***/.local/lib/python3.8/site-packages (from SQLAlchemy>=1.4->geoalchemy2) (2.0.2)
[2023-08-30T19:50:49.066+0000] {subprocess.py:93} INFO - Installing collected packages: sodapy, geoalchemy2
[2023-08-30T19:50:49.195+0000] {subprocess.py:93} INFO - Successfully installed geoalchemy2-0.14.1 sodapy-2.2.0
[2023-08-30T19:50:49.649+0000] {subprocess.py:93} INFO - 
[2023-08-30T19:50:49.650+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T19:50:49.651+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T19:50:49.937+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:50:49.992+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230825T000000, start_date=20230830T195005, end_date=20230830T195049
[2023-08-30T19:50:50.063+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:50:50.122+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:54:01.070+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:54:01.082+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-25T00:00:00+00:00 [queued]>
[2023-08-30T19:54:01.091+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-25T00:00:00+00:00 [queued]>
[2023-08-30T19:54:01.093+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:54:01.107+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-25 00:00:00+00:00
[2023-08-30T19:54:01.115+0000] {standard_task_runner.py:57} INFO - Started process 201 to run task
[2023-08-30T19:54:01.147+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-25T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmppfx70aqj']
[2023-08-30T19:54:01.163+0000] {standard_task_runner.py:85} INFO - Job 3: Subtask install_required_packages
[2023-08-30T19:54:01.213+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:54:01.325+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-25T00:00:00+00:00 [running]> on host 86f9e97d6c70
[2023-08-30T19:54:01.365+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:54:01.497+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-25T00:00:00+00:00'
[2023-08-30T19:54:01.500+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:54:01.502+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy geoalchemy2']
[2023-08-30T19:54:01.516+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:54:04.578+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T19:54:05.173+0000] {subprocess.py:93} INFO - Collecting sodapy
[2023-08-30T19:54:05.272+0000] {subprocess.py:93} INFO -   Downloading sodapy-2.2.0-py2.py3-none-any.whl (15 kB)
[2023-08-30T19:54:05.331+0000] {subprocess.py:93} INFO - Collecting geoalchemy2
[2023-08-30T19:54:05.624+0000] {subprocess.py:93} INFO -   Downloading GeoAlchemy2-0.14.1-py3-none-any.whl (54 kB)
[2023-08-30T19:54:05.670+0000] {subprocess.py:93} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 55.0/55.0 kB 1.3 MB/s eta 0:00:00
[2023-08-30T19:54:05.679+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T19:54:05.686+0000] {subprocess.py:93} INFO - Requirement already satisfied: SQLAlchemy>=1.4 in /home/***/.local/lib/python3.8/site-packages (from geoalchemy2) (1.4.49)
[2023-08-30T19:54:05.688+0000] {subprocess.py:93} INFO - Requirement already satisfied: packaging in /home/***/.local/lib/python3.8/site-packages (from geoalchemy2) (23.1)
[2023-08-30T19:54:05.701+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T19:54:05.704+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T19:54:05.706+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T19:54:05.709+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T19:54:05.759+0000] {subprocess.py:93} INFO - Requirement already satisfied: greenlet!=0.4.17 in /home/***/.local/lib/python3.8/site-packages (from SQLAlchemy>=1.4->geoalchemy2) (2.0.2)
[2023-08-30T19:54:22.143+0000] {subprocess.py:93} INFO - Installing collected packages: sodapy, geoalchemy2
[2023-08-30T19:54:22.597+0000] {subprocess.py:93} INFO - Successfully installed geoalchemy2-0.14.1 sodapy-2.2.0
[2023-08-30T19:54:23.069+0000] {subprocess.py:93} INFO - 
[2023-08-30T19:54:23.071+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T19:54:23.073+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T19:54:23.391+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:54:23.457+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230825T000000, start_date=20230830T195401, end_date=20230830T195423
[2023-08-30T19:54:23.509+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:54:23.549+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T20:05:25.359+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:05:25.373+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-25T00:00:00+00:00 [queued]>
[2023-08-30T20:05:25.381+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-25T00:00:00+00:00 [queued]>
[2023-08-30T20:05:25.382+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T20:05:25.406+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-25 00:00:00+00:00
[2023-08-30T20:05:25.411+0000] {standard_task_runner.py:57} INFO - Started process 413 to run task
[2023-08-30T20:05:25.420+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-25T00:00:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpypf9wzzz']
[2023-08-30T20:05:25.427+0000] {standard_task_runner.py:85} INFO - Job 8: Subtask install_required_packages
[2023-08-30T20:05:25.476+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:05:25.554+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-25T00:00:00+00:00 [running]> on host 3938d7abd9ff
[2023-08-30T20:05:25.580+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:05:28.491+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-25T00:00:00+00:00'
[2023-08-30T20:05:28.503+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T20:05:28.506+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy geoalchemy2']
[2023-08-30T20:05:28.525+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T20:05:38.325+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T20:05:41.618+0000] {subprocess.py:93} INFO - Requirement already satisfied: sodapy in /home/***/.local/lib/python3.8/site-packages (2.2.0)
[2023-08-30T20:05:41.621+0000] {subprocess.py:93} INFO - Requirement already satisfied: geoalchemy2 in /home/***/.local/lib/python3.8/site-packages (0.14.1)
[2023-08-30T20:05:41.623+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T20:05:41.625+0000] {subprocess.py:93} INFO - Requirement already satisfied: packaging in /home/***/.local/lib/python3.8/site-packages (from geoalchemy2) (23.1)
[2023-08-30T20:05:41.627+0000] {subprocess.py:93} INFO - Requirement already satisfied: SQLAlchemy>=1.4 in /home/***/.local/lib/python3.8/site-packages (from geoalchemy2) (1.4.49)
[2023-08-30T20:05:41.636+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T20:05:41.638+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T20:05:41.639+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T20:05:41.640+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T20:05:41.694+0000] {subprocess.py:93} INFO - Requirement already satisfied: greenlet!=0.4.17 in /home/***/.local/lib/python3.8/site-packages (from SQLAlchemy>=1.4->geoalchemy2) (2.0.2)
[2023-08-30T20:05:45.558+0000] {subprocess.py:93} INFO - 
[2023-08-30T20:05:45.562+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T20:05:45.569+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T20:05:45.821+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T20:05:45.904+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230825T000000, start_date=20230830T200525, end_date=20230830T200545
[2023-08-30T20:05:45.941+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T20:05:45.964+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T20:33:19.750+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:33:19.770+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-25T00:00:00+00:00 [queued]>
[2023-08-30T20:33:19.781+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-25T00:00:00+00:00 [queued]>
[2023-08-30T20:33:19.784+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T20:33:19.803+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-25 00:00:00+00:00
[2023-08-30T20:33:19.811+0000] {standard_task_runner.py:57} INFO - Started process 232 to run task
[2023-08-30T20:33:19.820+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-25T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpyz99pecb']
[2023-08-30T20:33:19.834+0000] {standard_task_runner.py:85} INFO - Job 6: Subtask install_required_packages
[2023-08-30T20:33:19.885+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:33:19.987+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-25T00:00:00+00:00 [running]> on host 961f38e8363e
[2023-08-30T20:33:20.024+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:33:20.247+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-25T00:00:00+00:00'
[2023-08-30T20:33:20.284+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T20:33:20.303+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy geoalchemy2']
[2023-08-30T20:33:20.336+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T20:33:33.572+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T20:33:39.261+0000] {subprocess.py:93} INFO - Requirement already satisfied: sodapy in /home/***/.local/lib/python3.8/site-packages (2.2.0)
[2023-08-30T20:33:39.263+0000] {subprocess.py:93} INFO - Requirement already satisfied: geoalchemy2 in /home/***/.local/lib/python3.8/site-packages (0.14.1)
[2023-08-30T20:33:39.270+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T20:33:39.275+0000] {subprocess.py:93} INFO - Requirement already satisfied: packaging in /home/***/.local/lib/python3.8/site-packages (from geoalchemy2) (23.1)
[2023-08-30T20:33:39.277+0000] {subprocess.py:93} INFO - Requirement already satisfied: SQLAlchemy>=1.4 in /home/***/.local/lib/python3.8/site-packages (from geoalchemy2) (1.4.49)
[2023-08-30T20:33:39.284+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T20:33:39.285+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T20:33:39.287+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T20:33:39.288+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T20:33:39.342+0000] {subprocess.py:93} INFO - Requirement already satisfied: greenlet!=0.4.17 in /home/***/.local/lib/python3.8/site-packages (from SQLAlchemy>=1.4->geoalchemy2) (2.0.2)
[2023-08-30T20:33:45.960+0000] {subprocess.py:93} INFO - 
[2023-08-30T20:33:45.969+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T20:33:45.971+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T20:33:46.656+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T20:33:46.692+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230825T000000, start_date=20230830T203319, end_date=20230830T203346
[2023-08-30T20:33:46.735+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T20:33:46.768+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T20:48:14.470+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:48:14.491+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-25T00:00:00+00:00 [queued]>
[2023-08-30T20:48:14.507+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-25T00:00:00+00:00 [queued]>
[2023-08-30T20:48:14.509+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T20:48:14.546+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-25 00:00:00+00:00
[2023-08-30T20:48:14.551+0000] {standard_task_runner.py:57} INFO - Started process 288 to run task
[2023-08-30T20:48:14.562+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-25T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmp2nmuvvf2']
[2023-08-30T20:48:14.575+0000] {standard_task_runner.py:85} INFO - Job 5: Subtask install_required_packages
[2023-08-30T20:48:14.605+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:48:14.677+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-25T00:00:00+00:00 [running]> on host 8f66711a2898
[2023-08-30T20:48:14.698+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:48:14.825+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-25T00:00:00+00:00'
[2023-08-30T20:48:14.827+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T20:48:14.829+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy']
[2023-08-30T20:48:14.846+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T20:48:16.391+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T20:48:16.887+0000] {subprocess.py:93} INFO - Requirement already satisfied: sodapy in /home/***/.local/lib/python3.8/site-packages (2.2.0)
[2023-08-30T20:48:16.892+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T20:48:16.902+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T20:48:16.907+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T20:48:16.909+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T20:48:16.912+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T20:48:21.199+0000] {subprocess.py:93} INFO - 
[2023-08-30T20:48:21.201+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T20:48:21.202+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T20:48:21.410+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T20:48:21.460+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230825T000000, start_date=20230830T204814, end_date=20230830T204821
[2023-08-30T20:48:21.488+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T20:48:21.526+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T21:07:24.054+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:07:24.063+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-25T00:00:00+00:00 [queued]>
[2023-08-30T21:07:24.076+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-25T00:00:00+00:00 [queued]>
[2023-08-30T21:07:24.078+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T21:07:24.096+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-25 00:00:00+00:00
[2023-08-30T21:07:24.138+0000] {standard_task_runner.py:57} INFO - Started process 262 to run task
[2023-08-30T21:07:24.157+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-25T00:00:00+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmp_6cmigd2']
[2023-08-30T21:07:24.174+0000] {standard_task_runner.py:85} INFO - Job 7: Subtask install_required_packages
[2023-08-30T21:07:24.238+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:07:24.337+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-25T00:00:00+00:00 [running]> on host cbc745b7550b
[2023-08-30T21:07:24.352+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:07:24.457+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-25T00:00:00+00:00'
[2023-08-30T21:07:24.475+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T21:07:24.478+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy']
[2023-08-30T21:07:24.499+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T21:07:26.670+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T21:07:27.054+0000] {subprocess.py:93} INFO - Requirement already satisfied: sodapy in /home/***/.local/lib/python3.8/site-packages (2.2.0)
[2023-08-30T21:07:27.057+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T21:07:27.064+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T21:07:27.067+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T21:07:27.068+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T21:07:27.069+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T21:07:33.432+0000] {subprocess.py:93} INFO - 
[2023-08-30T21:07:33.434+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T21:07:33.435+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T21:07:33.642+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T21:07:33.669+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230825T000000, start_date=20230830T210724, end_date=20230830T210733
[2023-08-30T21:07:33.731+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T21:07:33.754+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T21:16:36.723+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:16:36.737+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-25T00:00:00+00:00 [queued]>
[2023-08-30T21:16:36.757+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-25T00:00:00+00:00 [queued]>
[2023-08-30T21:16:36.758+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T21:16:36.819+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-25 00:00:00+00:00
[2023-08-30T21:16:36.825+0000] {standard_task_runner.py:57} INFO - Started process 302 to run task
[2023-08-30T21:16:36.843+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-25T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpdbmkezbd']
[2023-08-30T21:16:36.859+0000] {standard_task_runner.py:85} INFO - Job 4: Subtask install_required_packages
[2023-08-30T21:16:36.909+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:16:36.979+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-25T00:00:00+00:00 [running]> on host 5b9f14bf20e0
[2023-08-30T21:16:37.002+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:16:37.101+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-25T00:00:00+00:00'
[2023-08-30T21:16:37.135+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T21:16:37.138+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy']
[2023-08-30T21:16:37.162+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T21:16:48.180+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T21:16:50.288+0000] {subprocess.py:93} INFO - Requirement already satisfied: sodapy in /home/***/.local/lib/python3.8/site-packages (2.2.0)
[2023-08-30T21:16:50.295+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T21:16:50.298+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T21:16:50.299+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T21:16:50.300+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T21:16:50.301+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T21:16:59.912+0000] {subprocess.py:93} INFO - 
[2023-08-30T21:16:59.922+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T21:16:59.926+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T21:17:00.288+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T21:17:00.424+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230825T000000, start_date=20230830T211636, end_date=20230830T211700
[2023-08-30T21:17:00.497+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T21:17:00.565+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
